newdle/core/app.py

```diff
@@ -81,6 +83,34 @@ def _handle_exception(exc):
         return jsonify(error='internal_error'), 500
 
 
+def _configure_cli(app):
+    @app.cli.command('cleanup_newdles')
```

Comment:
I'd prefer to have the actual implementation elsewhere, e.g. in a `cli.py`. Easiest option to do it would be adding a `cli` blueprint so you can define the cli functions there and then just register the blueprint in here.